// Copyright (c) 2010 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for autofill.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package sync_pb;

import "sync.proto";

// Properties of autofill sync objects.
message AutofillSpecifics {
  optional string name = 1;
  optional string value = 2;
  repeated int64 usage_timestamp = 3;
}

extend EntitySpecifics {
  optional AutofillSpecifics autofill = 31729;
}
